examples/tests/valid/optionals.test.w

```diff
@@ -204,3 +204,13 @@ if let s1 = str1 {
 } elif let s2 = str2 {
   assert(true);
 }
+
+let functionReturningNothing = (): void? => {
```

Comment:
i thought the whole idea is that `void` is implicitly an optional, no?

```suggestion
let functionReturningNothing = (): void => {
```

Reply:
My idea was, if you want to check that returning ```void``` (doing ```return;```) in a function is ok in a function that expects an optional value (```void?```, ```num?```, etc.), then this test would verify that. 

Technically ```void?``` and ```void``` lead to the same case in ```is_subtype_of()``` being triggered, but I thought this might be easier to read and make the idea more explicit. Let me know if this should be changed though and I'll update the PR.

Reply:
I see, so let's change this to `num?` or `str?` instead.